### PR TITLE
Snapshot rst

### PR DESCRIPTION
**Pull request by Darcy-Linde**

### CODE REVIEW

File: docs/source/Snapshot.rst [hunk 1]
```diff
@@ -0,0 +1,29 @@
+.. _trades_header:
```
Comment (clickingbuttons) [inline docs/source/Snapshot.rst:1]: snapshot_header

**Review by clickingbuttons — CHANGES_REQUESTED** [review]
Add "Snapshot" to index.rst

File: docs/source/Snapshot.rst [hunk 2]
```diff
@@ -0,0 +1,29 @@
+.. _snapshot_header:
+
+Snapshot
+=============
+
+===========
```
Comment (clickingbuttons) [inline docs/source/Snapshot.rst:6]: It's annoying, but the docs throw a warning unless these are as long as the title. Can you make them longer?

**Review by clickingbuttons — CHANGES_REQUESTED** [review]

File: docs/source/Snapshot.rst (hunk 2 above)
Comment (Darcy-Linde) [reply docs/source/Snapshot.rst:6]: Sure thing

**Review by clickingbuttons — APPROVED** [review]

**Pull request merged**